lips/lip-6.md

```diff
@@ -0,0 +1,79 @@
+
+# Specification
+## Overview
+Each Validator Owner holds a Libra account with a *Validator Owner role* (see [lip-2: Libra Roles and Permissions](https://github.com/libra/lip/blob/master/lips/lip-2.md)). A Validator Owner designates a Validator Operator to run a Validator Node
+A Validator Owner designates a single Validator Operator account to run a Validator Node. Validator Operator is a Libra account with a *Validator Operator role*. Each Validator Operator may run nodes for multiple Validator Owners, therefore there is a one-to-many relationship between operator accounts and owner accounts. If an operator OP_1 is designated to run Validator Nodes for two Validator Owners V_1 and V_2, the operator must run two distinct instances of Validator Nodes.
+
+Each Validator Owner account stores a consensus configuration that consists of the consensus public key, the networking information (for the Validator Node to connect to other Validator Nodes) and the Full Node networking information (for the public Full Nodes and clients and, under certain circumstances, for other Validator Nodes themselves to connect to this Validator Node). The Validator Operator designated by the Validator Owner is responsible for running a node with this consensus config and is responsible for updating/rotating this config as needed.
+
+The Validator Set is a map of Validator Owner addresses to a copy of their consensus configs. Each Validator Owner address appears in the Validator Set at most once. This set is stored under the LibraRoot address and reflects the configuration that currently runs consensus protocol. Any change to this set defines a new epoch by triggering a new epoch event. If a transaction causes an epoch change, that transaction is the last transaction committed in the current epoch — any subsequent transactions in that block or future blocks from that epoch will be ignored. Once the transaction has been committed, the new Validator Set can start the next epoch of the consensus protocol.
+
+Only LibraRoot may add/remove Validators Owner accounts to/from the Validator Set. Each Validator Operator may update its validator's config in this set which will also trigger the a new epoch event.
+
+Each Validator Owner is obliged to notify LN about the decision to change the Validator Operator, LN may disallow the switch if the new operator is deemed not trustworthy or the distribution of consensus power becomes skewed towards this operator or operators with similar architectures. LN makes sure there is enough diversity between the Validator Nodes operating the Libra Blockchain.
+
+## Move Modules
+
+Four move modules are involved in the validator configuration management
+
+* [`ValidatorConfig`](https://github.com/libra/libra/blob/master/language/stdlib/modules/ValidatorConfig.move): stores local consensus configuration for a validator.
```

Comment:
More precisely, it defines structures (resources?) to hold config information for validators.   It doesn't actually publish this info.